src/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 100px auto;
+  width: 700px;
+  height: 700px;
+  position: relative;
+}
+
+.center {
+  width: 36px;
+  height: 36px;
+  border-radius: 18px;
+  background: #0095b6;
+  position: absolute;
+  top: 390px;
+  left: 338px;
+}
+
+.seconds {
+  width: 10px;
+  height: 400px;
+  border-radius: 5px;
+  background: #0095b6;
+  position: absolute;
+  left: 350px;
+  top: 15px;
+  animation: revolve 60s linear 5;
+  transform-origin: bottom;
+}
+
+.minutes {
+  width: 12px;
+  height: 300px;
+  border-radius: 5px;
+  background: red;
+  position: absolute;
+  left: 350px;
+  top: 115px;
+  animation: revolve-min 3600s linear;
+  transform-origin: bottom;
+}
+
+@keyframes revolve {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+
+}
+
+@keyframes revolve-min {
```

Comment:
duplicated, you can use revolve

Reply:
Done). I was so happy that it moves, that even didn't think about duplications )